cypress/e2e/cardFeatures.js

```diff
@@ -88,7 +88,6 @@ describe('Card', function () {
 		cy.wait('@save', { timeout: 7000 })
 
 		cy.visit(`/apps/deck/#/board/${boardId}`)
-		cy.reload()
```

Comment:
Was this check not intended to double check that the card indeed exists even after a reload? Otherwise it could (theoretically) be that there is only a presentation, but the backend actually did not store anything.

Reply:
I think you are right... let me try to find another solution.